bin/buildNS.bash

```diff
@@ -1,12 +0,0 @@
-#!/usr/bin/env bash
-
-cd ./build || exit
-
-jsFiles=$(find . -name "*.js")
-
-echo "$jsFiles" | while read -r line; do
-  echo "> Building file ${line%.*}.ns"
-  sed -re 's/(import.*\"\/.*)\"/\1\.ns\"/g' "$line" > "${line%.*}.ns"
```

Comment:
This was used to make sure the import was in absolute form instead of relative because relative paths did not work. Do they work now? If not, keep this file, just replace ns with js; the script will only make sure path are absolute and not change the file extension :)

Reply:
You're absolutely correct! My bad :)

Reply:
Wait, actually, I am weirdly finding that buildNS.bash is causing my build/ files to be empty... Do you have any idea what could be causing this? I am going to remove it again because I am experiencing issues when it is in. I will keep playing around and let you know what I find

Reply:
Hmm, what OS are you running on ?

Reply:
PopOS, but I can also try it on WSL Ubuntu